zerver/lib/test_helpers.py

```diff
@@ -280,21 +308,23 @@ def send_message(self, sender_name, recipient_list, message_type,
             forwarder_user_profile=sender, realm=sender.realm, **kwargs)
 
     def get_old_messages(self, anchor=1, num_before=100, num_after=100):
+        # type: (int, int, int) -> Iterable[Dict[str, Any]]
```

Comment:
I think the return value of this is most precisely a List (though it is also an Iterable).